src/bzflag/Player.cxx

```diff
@@ -112,10 +112,7 @@ Player::Player(const PlayerId& _id, TeamColor _team, int _skinIndex, const char*
         tankIDLNode = new TankIDLSceneNode(tankNode);
         changeTeam(team);
         const float sphereRad = (1.5f * BZDBCache::tankRadius);
-        if (RENDERER.useQuality() >= 2)
-            pausedSphere = new SphereLodSceneNode(state.pos, sphereRad);
-        else
-            pausedSphere = new SphereBspSceneNode(state.pos, sphereRad);
+        pausedSphere = new SphereSceneNode(state.pos, sphereRad);
```

Comment:
as an aside, there is an initialization problem here; if this branch isn't taken `pausedSphere` is not initialized. You didn't cause it but...

Reply:
Initialized with nullptr